src/esp/assets/ResourceManager.cpp

```diff
@@ -1607,9 +1608,15 @@ gfx::PhongMaterialData::uptr ResourceManager::buildFlatShadedMaterialData(
     // check for diffuse texture and use that instead
     finalMaterial->ambientTexture =
         textures_.at(textureBaseIndex + material.diffuseTexture()).get();
-  } else {
+  }
+
+  if (material.hasAttribute(Mn::Trade::MaterialAttribute::DiffuseColor)) {
+    finalMaterial->ambientColor = material.diffuseColor();
+  } else if (material.hasAttribute(
+                 Mn::Trade::MaterialAttribute::AmbientColor)) {
     finalMaterial->ambientColor = material.ambientColor();
   }
```

Comment:
If it does NOT have either the diffuse color or the ambient color, it will set the `finalMaterial->ambientColor = Mn::Vector3(1.0f, 1.0f, 1.0f)`.

Do you think this is necessary?

Reply:
I agree with you and @eundersander : If there is no ambient color, the mesh will not be visible. 

See the code here:
https://github.com/mosra/magnum/blob/dd3ce93888ada70074d63d72c5998d950fa2eba6/src/Magnum/Shaders/Phong.frag#L250

Reply:
The alternative is to set it to `material.ambientColor()`. If `material` has an ambient texture, this will set it to white, which is the same result. Otherwise, it will be set to black. I think it's more useful for it to be set to white so that vertex colored meshes are viewable.
<img width="529" alt="Screen Shot 2021-06-04 at 2 53 17 PM" src="https://user-images.githubusercontent.com/4172272/120866385-ac0e5300-c544-11eb-935f-93e852285f57.png">

Reply:
I did not understand your comment. Our goal is to set the `finalMaterial` and `material` is just the input.

What I suggested is:

```
  if (material.hasAttribute(Mn::Trade::MaterialAttribute::DiffuseColor)) {
    finalMaterial->ambientColor = material.diffuseColor();
  } else if (material.hasAttribute(
                 Mn::Trade::MaterialAttribute::AmbientColor)) {
    finalMaterial->ambientColor = material.ambientColor();
  } else {
  // set finalMaterial explicitly to some color.e.g, (1.0f, 1.0f, 1.0f);
}
```
If there is NO light, yes, we need to set the ambientColor in the final Material. Cannot let it black. Otherwise you will see nothing.

Reply:
Oh I see what you mean. I think the code already does this though on the previous line
```  finalMaterial->ambientColor = 0xffffffff_rgbaf;```
If `material` has neither diffuseColor nor ambientColor, then `finalMaterial->ambientColor` will stay as white.

Reply:
Oh, OK. I see. 
`finalMaterial->ambientColor = 0xffffffff_rgbaf;`
This is hided by the github.

Then I do not have any questions. Make sure the CI tests passed. Then merge it. Thanks.